### PR TITLE
Add ParseTest

### PR DESCRIPTION
**Pull request by kwokyto**

### CODE REVIEW

**Review by Leeyp — APPROVED** [review]
Looks good. Can consider adding more JUnit tests for future iterations.

**Review by SimJJ96 — APPROVED** [review]
LGTM

File: src/test/java/seedu/fridgefriend/utilities/ParserTest.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    public void parse_addCommandNoArgs_EmptyDescriptionException() {
+        assertThrows(EmptyDescriptionException.class, () -> {
+            Parser.getCommand("add");
+        });
+    }
+
+    @Test
+    public void parse_addCommandMissingArgs_InvalidInputException() {
+        assertThrows(InvalidInputException.class, () -> {
+            Parser.getCommand("add /c meat");
```
Comment (Vinci-Hu) [inline src/test/java/seedu/fridgefriend/utilities/ParserTest.java]: You changed this to /cat right?

**Review by Vinci-Hu — APPROVED** [review]
LGTM

**Pull request merged**